api/pom.xml

```diff
@@ -39,15 +40,15 @@
         <dependency>
             <groupId>org.osgi</groupId>
             <artifactId>org.osgi.annotation.versioning</artifactId>
-            <version>1.0.0</version>
+            <version>1.1.0</version>
         </dependency>
     </dependencies>
 
     <build>
         <plugins>
             <plugin>
                 <groupId>org.apache.maven.plugins</groupId>
-                <artifactId>maven-javadoc-plugin</artifactId>
+                <artifactId>maven-javadoc-plugin</artifactId>                
```

Comment:
```suggestion
                <artifactId>maven-javadoc-plugin</artifactId>
```

Reply:
done

Reply:
The GitHub diff still shows the extra space at the end of line?